style.css

```diff
@@ -0,0 +1,667 @@
+html {
+  -ms-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+}
+
+ul {
+  display: block;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  display: block;
+}
+
+body {
+  display: flex;
+  font-family: sans-serif;
+  line-height: 1.42857143;
+  color: rgb(238, 229, 229);
+  margin: 0;
+  padding: 0;
+  flex-direction: column;
+  align-items: flex-start;
+  position: absolute;
+  width: 375px;
+  height: 604px;
+  top: 0;
+  left: calc(50% - 375px / 2);
+  background-color: #e5e5e5;
+}
+
+.main-nav {
+  overflow: hidden;
+  background-color: #6070ff;
+  position: relative;
+  z-index: 1;
+  color: #f7f7f9;
+}
+
+.mylogo {
+  font-family: sans-serif;
+  font-weight: 900;
+  font-size: 20px;
+}
+
+.main-nav a {
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 18px;
+  display: block;
+}
+
+.main-nav a.icon {
+  display: block;
+  position: absolute;
+  color: #6070ff;
+  right: 0;
+  top: 20px;
+}
+
+.active {
+  background-color: #f7f7f9;
+  color: #6070ff;
+  font: bold;
+  font-size: 30px;
+  width: 375px;
+  height: 50px;
+  top: auto;
+  font-weight: 900px;
+}
+
+.main {
+  position: relative;
+  overflow: hidden;
+  min-height: 4810px;
+  flex-shrink: 0;
+  display: flex;
+  flex-direction: column;
+  background-color: #f7f7f9;
+}
+
+.nav-flex {
+  position: relative;
+  height: -webkit-min-content;
+  height: -moz-min-content;
+  height: min-content;
+  margin: 16px 15px;
+  display: flex;
+  flex-direction: column;
+  background: url(./img/backgroung_image.png) no-repeat;
+}
+
+.title-main {
+  display: flex;
+  align-items: center;
+  position: relative;
+  height: -webkit-min-content;
+  height: -moz-min-content;
+  height: min-content;
+  margin: 12px 9px 0;
+}
+
+.name-text {
+  overflow: visible;
+  margin-top: 80px;
+  margin-bottom: 0;
+  font: 700 40px/1.3 "Poppins", Helvetica, Arial, serif;
+  color: #172b4d;
+  letter-spacing: 0;
+  white-space: pre-wrap;
+}
+
+.sub-text {
+  position: relative;
+  height: -webkit-min-content;
+  height: -moz-min-content;
+  height: min-content;
+  margin-top: 35px 9px 0;
+  bottom: 50px;
+  font: 16px/1.5 "Poppins", Helvetica, Arial, serif;
+  color: #344563;
+  letter-spacing: 0;
+  font-weight: 400;
+}
+
+.mini-text {
+  position: relative;
+  height: -webkit-min-content;
+  height: -moz-min-content;
+  height: min-content;
+  margin: 14px 9px 0;
+  bottom: 40px;
+  font: 500 16px/1.5 "Poppins", Helvetica, Arial, serif;
+  color: #7f8cff;
+  letter-spacing: 0;
+}
+
+.social-media {
+  display: flex;
+  position: relative;
+  height: -webkit-min-content;
+  height: -moz-min-content;
+  height: min-content;
+  bottom: 40px;
+  margin: 12px 9px 120px;
+}
+
+.social-media-item {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  flex: 0 1 20px;
+}
+
+.media-icon {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 170.17px;
+  height: 20px;
+}
```

Comment:
Your social media icons don't look quite like the ones from the template design.

Here are your icons:

<img width="130" alt="Screen Shot 2022-05-27 at 17 02 01" src="https://user-images.githubusercontent.com/82426682/170794555-8562d595-66e8-40b0-b857-346be33f43e2.png">

Here are the template design icons:
<img width="241" alt="Screen Shot 2022-05-27 at 17 02 29" src="https://user-images.githubusercontent.com/82426682/170794611-2f0a8656-3407-4711-a1b2-ce110cfe4549.png">

As you can see, the order of the icons is not the same as in the template. The template doesn't have a Facebook icon, you're missing the Medium icon and the colors are not the same. I recommend you export the icons directly from Figma to avoid having this issue.